### ISSUE TITLE
Position for label on arrow documented but not implemented

### ISSUE THREAD

**Issue by Cyberwizzard** [drom/wavedrom#151]
From the syntax page at https://github.com/drom/wavedrom/wiki/WaveJSON :
"# - indicates the arrow label position"

I can not get this to work and looking at the code I also can't find any evidence that this is implemented. Or am I missing something?

**Comment by drom** [comment]
Yes, we never implemented proper arrows and labels. We probably should.
